packages/x-date-pickers/src/internals/hooks/usePicker/usePickerProvider.ts

```diff
@@ -67,6 +68,14 @@ export function usePickerProvider<TValue extends PickerValidValue>(
   const utils = useUtils();
   const orientation = usePickerOrientation(views, props.orientation);
 
+  const handleTogglePicker = useEventCallback((event: React.UIEvent) => {
```

Comment:
It could be an action exposed by `usePickerValue` but it has more implications (all actions are forwarded to `PickersPopper` for now.
Until the actions are always accessed through the state, I propose to keep it here to avoid problems.